packages/ember-routing-htmlbars/tests/helpers/closure_action_test.js

```diff
@@ -55,7 +54,7 @@ function registerComponent(name, factory) {
 }
 
 function appendViewFor(template, moduleName='', hash={}) {
-  let view = EmberView.extend({
+  let view = EmberComponent.extend({
     template: compile(template, { moduleName }),
```

Comment:
This needs to change to `layout` (a `template` property does nothing in a component).